docs/IntakeApi.md

```
@@ -23,9 +23,8 @@ Submit a new work item to the project's intake queue for review and triage. Auto
 * Api Key Authentication (ApiKeyAuthentication):
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)